constants.py

```diff
@@ -3,8 +3,20 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
```

Comment:
What is this empty lien for?

Reply:
Hmm... , in practical situation, the variables should be placed by dictionary order.
Thus, ROOT_DIR could not be put after DATA_DIR.
To put them in some kind of good order, I used a line to separate them make them as two parts.

Reply:
I see. Sounds reasonable.